api-reference/10 UI Components/dxTreeList/1 Configuration/autoExpandAll.md

```diff
@@ -8,6 +8,9 @@ default: false
 Specifies whether all rows are expanded initially.
 
 ---
+
+If you enable the **autoExpandAll** property, the TreeList ignores the [remoteOperations](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/remoteOperations/) property and loads all data to the client first. Filtering executes in JavaScript code.
```

Comment:
Pardon, filtering and sorting according to our code:
https://github.com/DevExpress/DevExtreme/blame/98968c5bf453d02500a8c0d337a0d12f671ae36b/js/__internal/grids/tree_list/data_source_adapter/module.ts#L236